guides/getting_started.md

```diff
@@ -131,6 +131,9 @@ It contains the list of components and systems to deploy, and which systems need
 This file is the source of truth for MUD's deployment tools.
 If a component or system is not included, it is not deployed, even if it is placed in the `components` or `systems` directory.
 
+`libs` are optional. After components and systems are deployed, libs will be called with `IWorld` as the sole argument, e.g. `LibInitData.init(world)`.
+Libs should be in the `libraries` directory.
```

Comment:
We should probably add more context to answer the "why" behind this: what is this useful for?